examples/Python/Utility/file.py

```diff
@@ -30,9 +29,16 @@ def get_file_list(path, extension=None):
 
 
 def add_if_exists(path_dataset, folder_names):
+    path = None
     for folder_name in folder_names:
```

Comment:
The original code looks pretty strange -- if there is more than one existing `folder_names`, only the last one will be picked up. We shall return the path if and only if there is exactly one `folder_name` inside `folder_names` exists.

So probably something like this
```python
valid_paths = [join(path_dataset, f) for f in folder_names if exists(join(path_dataset, f))]
if len(valid_paths) == 0:
    # exception not found
elif len(valid_paths) > 1:
     # exception more than one found
else:
    return valid_paths[0]
```